helix-term/src/commands/typed.rs

```diff
@@ -1406,21 +1420,32 @@ fn lsp_workspace_command(
         cx.jobs.callback(callback);
     } else {
         let command = args.join(" ");
```

Comment:
I am unsure about this one. Is there some LSP which has a space in the command name. I would personally like this to be such that the first argument is the command and the remaining ones are arguments to that command. But I have kept it as it was for now.